libcloud/dns/drivers/powerdns.py

```diff
@@ -143,7 +144,7 @@ def __init__(self, key, secret=None, secure=False, host=None, port=None,
                                              host=host, port=port,
                                              **kwargs)
 
-    def create_record(self, name, zone, type, data, extra=None):
+    def create_record(self, name, zone, type, data, extra=None, disabled=False, comment=None):
```

Comment:
Those are extension arguments (arguments which are not supported by all the providers), please prefix them with ``ex_``.

So ``ex_disabled`` and ``ex_comment``.